source/mainnet/net/guides/deciding-wallet.rst

```diff
@@ -1,35 +1,39 @@
 
-All of the wallets are developed by Concordium Software and can only be used to hold :ref:`CCD<glossary-CCD>`, the native token of the Concordium blockchain. You can't use these wallets for any other cryptocurrencies.
+The wallets are developed by Concordium Software and can only be used to hold :ref:`CCD<glossary-CCD>`, the native token of the Concordium blockchain. You can't use these wallets for any other cryptocurrencies.
 
 No import of identities and accounts from one wallet to another
 ===============================================================
 
-Before you decide which wallet to use, it's important to know that you can’t import identities and accounts from the |mw-gen1| or |mw-gen2| into the Desktop Wallet. Conversely, you can’t import identities and accounts from the Desktop Wallet into |mw-gen1| or |mw-gen2|. You also can't import identities and accounts between |mw-gen1| and |mw-gen2|. This is because they handle private keys in different ways as :ref:`described later<store-private-keys>` in this topic. It’s therefore essential that you choose between the wallets before you start creating identities and accounts.
+Before you decide which wallet to use, it's important to know that you can’t import identities and accounts from the |mw-gen1|, |mw-gen2|, or |bw| into the Desktop Wallet. Conversely, you can’t import identities and accounts from the Desktop Wallet into |mw-gen1|, |mw-gen2|, or |bw|. You also can't import identities and accounts between |mw-gen1| and |mw-gen2|. This is because they handle private keys in different ways as :ref:`described later<store-private-keys>` in this topic. It’s therefore essential that you choose between the wallets before you start creating identities and accounts.
 
-.. warning:: You can't exchange identities and accounts between |mw-gen1| and |mw-gen2|, or |mw-gen1| or |mw-gen2| and the Desktop Wallet. You can, however, send CCD from one wallet to another.
+You can only exchange identities and accounts between the |bw| and the |mw-gen2|.
+
+You can always send CCD from one wallet to another.
 
 Which wallet is best for you?
 =============================
 
-The wallets work on two different environments: the Desktop Wallet on a computer, and |mw-gen1| and |mw-gen2| on a phone. The |mw-gen1| and |mw-gen2| are more accessible and easier to use on a day-to-day basis whereas the Desktop Wallet has an extra layer of security in the form of a Ledger device.
+The wallets work on two different environments: the Desktop Wallet on a computer, the |bw| on any device with a web browser, and |mw-gen1| and |mw-gen2| on a phone. The |mw-gen1| and |mw-gen2| are more accessible and easier to use on a day-to-day basis whereas the Desktop Wallet has an extra layer of security in the form of a Ledger device.
```

Comment:
any device with a web browser includes a phone, but I don't think that is the case (at least not without using some other chromium browser)